api/controllers/restaurantController.js

```diff
@@ -10,41 +10,42 @@ const createRestaurant = async (req, res) => {
 };
 
 const getRestaurants = async (req, res) => {
-  console.log('12', req.query);
-  // TODO: add category
   const {
-    category = 'created_at', order = 'DESC', cuisine = 2, limit = 10, search = '',
+    category = 'created_at', order = 'DESC', cuisine, limit = 10, search, tags,
   } = req.query;
-  // const { category, order, cuisine } = req.query;
-  console.log('lin 16', category, order, cuisine);
-  // const restaurants = await Restaurant.findAll({
-  //   where: {
-  //     isVisible: true,
-  //   },
-  //   order: [
-  //     [category, order],
-  //   ],
-  //   limit,
-  //   include: ['cuisines', 'reviews'],
-  // });
-
   const restaurants = await Restaurant.findAll({
     where: {
-      [Op.or]: [
-        { name: { [Op.like]: `%${search}%` } },
-      ],
+      isVisible: true,
+      ...(search ? {
+        name: {
+          [Op.iLike]: `%${search}%`,
```

Comment:
Not sure what 'Op' is for, i'm assuming this is the db model name? No immediate changes required but consider using a more descriptive name in the future

Reply:
Op is a sequelize method for the query constants